packages/app/src/debug/DebugSpec.vue

```diff
@@ -78,73 +122,140 @@
+        :class="Object.keys(specData.groups).length > 1 ? 'pb-16px': 'hover:bg-gray-50'"
+      >
+        <DebugFailedTest
+          :failed-test-result="specData.failedTests[thumbprint]"
+          :groups="groupsPerTest[thumbprint]"
+          :expandable="Object.keys(specData.groups).length > 1"
+        />
+      </div>
     </div>
   </div>
 </template>
 <script lang="ts" setup>
+
 export interface Spec {
   id: string
   path: string
   fileName: string
   fileExtension: string
+  testsPassed: SpecDataAggregate
+  testsFailed: SpecDataAggregate
+  testsPending: SpecDataAggregate
+  specDuration: SpecDataAggregate
   fullPath: string
 }
 
 export interface TestResults {
   readonly id: string
-  readonly titleParts: ReadonlyArray<string>
+  readonly titleParts: string[]
+  instance: CloudRunInstance
 }
 
 import { computed } from 'vue'
-import { IconActionRefresh } from '@cypress-design/vue-icon'
+import { IconActionRefresh, IconDocumentText } from '@cypress-design/vue-icon'
+import type { SpecDataAggregate, CloudRunInstance, CloudRunGroup } from '@packages/data-context/src/gen/graphcache-config.gen'
 import DebugFailedTest from './DebugFailedTest.vue'
+import StatsMetaData from './StatsMetadata.vue'
+import ResultCounts from '@packages/frontend-shared/src/components/ResultCounts.vue'
 import Button from '@packages/frontend-shared/src/components/Button.vue'
 import Tooltip from '@packages/frontend-shared/src/components/Tooltip.vue'
 import SpecNameDisplay from '../specs/SpecNameDisplay.vue'
 import { useI18n } from '@cy/i18n'
+import { useDurationFormat } from '../composables/useDurationFormat'
 import { posixify } from '../paths'
 import type { TestingTypeEnum } from '../generated/graphql'
 
 const { t } = useI18n()
 
 const props = defineProps<{
   spec: Spec
-  testResults: TestResults[]
-  foundLocally: boolean
+  testResults: {[thumbprint: string]: TestResults[]}
+  groups: {[groupId: string]: CloudRunGroup }
   testingType: TestingTypeEnum
+  isDisabled?: boolean
+  foundLocally: boolean
   matchesCurrentTestingType: boolean
 }>()
 
 const emits = defineEmits<{
   (event: 'switchTestingType', value: TestingTypeEnum): void
 }>()
 
+// helper function for formatting the number of passed, failed, and pending tests
+const debugResultsCalc = (min: number, max: number, specDuration: boolean = false) => {
+  if (min === max) {
+    return specDuration ? useDurationFormat(min ?? 0).value : min
+  }
+
+  return specDuration ? `${useDurationFormat(min ?? 0).value}-${useDurationFormat(max ?? 0).value}` : `${min}-${max}`
+}
+
 const specData = computed(() => {
+  const testsPassed = props.spec.testsPassed ?? 0
+  const testsFailed = props.spec.testsFailed ?? 0
+  const testsPending = props.spec.testsPending ?? 0
+  const specDuration = props.spec.specDuration ?? 0
+
   return {
-    path: props.spec.path,
+    path: `${props.spec.path}/`,
```

Comment:
An extra slash is getting rendered when real data is being used:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/1702361/209699255-0575bab6-5b94-4388-876f-932b87fc32bd.png">